docs/paas-plus/dev/check-spryker-code-upgrader-logs.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Check Spryker Code Upgrader logs
+description: Instructions for checking Spryker Code Upgrader logs
+template: howto-guide-template
+---
+
+To check the logs of Spryker Code Upgrader, do the following:
+
+1. In Spryker CI, go to **Pipelines**.
```

Comment:
I would say: 

1. In Spryker CI workspace go to **Projects**.
2. Select **Pipelines** menu item.

Reply:
or `Pick the **Pipelines** menu item on the Spryker CI workspace page.`